manifests/config.pp

```diff
@@ -133,4 +133,66 @@
       }
     }
   }
+
+  if $foreman_proxy::manage_certificates {
+    file { "${foreman_proxy::config_dir}/ssl_ca.pem":
+      ensure => file,
+      source => $foreman_proxy::ssl_ca,
+      owner  => $foreman_proxy::user,
+      group  => $foreman_proxy::group,
+      mode   => '0440',
+      notify => Service['foreman-proxy'],
```

Comment:
I'd think that any chance in this config class refreshes the service and this should be redundant.